### PR TITLE
=act #18346 Document scheduler task throws

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #18346

**Comment by patriknw** [comment]
failure was unrelated https://github.com/akka/akka/issues/17314

**Comment by patriknw** [comment]
@drewhk please review

File: akka-actor/src/main/scala/akka/actor/Scheduler.scala [hunk 1]
```diff
@@ -74,6 +74,9 @@ trait Scheduler {
    * of the function executions). In such cases, the actual execution interval
    * will differ from the interval passed to this method.
    *
+   * If the function throws an exception the repeated scheduling is aborted,
```
Comment (drewhk) [inline akka-actor/src/main/scala/akka/actor/Scheduler.scala]: That's an important information :)
It is logged at error level though, right?

Comment (drewhk) [inline akka-actor/src/main/scala/akka/actor/Scheduler.scala]: this raises the question of who timeouts on the timeouts ;)

**Comment by drewhk** [comment]
LGTM

File: akka-actor/src/main/scala/akka/actor/Scheduler.scala (hunk 1 above)
Comment (patriknw) [inline akka-actor/src/main/scala/akka/actor/Scheduler.scala]: It is logged. In the test I added:

    [ERROR] [09/16/2015 11:40:12.744] [LightArrayRevolverSchedulerSpec-akka.actor.default-dispatcher-3] [TaskInvocation] TEST (akka.actor.SchedulerSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$apply$mcV$sp$5$$anon$1)

I think it is logged via the `ExecutionContext.reportFailure` and for our dispatchers we publish that Error to the eventStream

**Comment by ktoso** [comment]
LGTM

**Pull request merged**